sdk/bpf/scripts/objcopy.sh

```diff
@@ -3,4 +3,4 @@
 bpf_sdk=$(cd "$(dirname "$0")/.." && pwd)
 # shellcheck source=sdk/bpf/env.sh
 source "$bpf_sdk"/env.sh
-exec "$bpf_sdk"/dependencies/llvm-native/bin/llvm-objcopy "$@"
+exec "$bpf_sdk"/dependencies/bpf-tools/llvm/bin/llvm-objcopy "$@"
```

Comment:
```suggestion
exec "$bpf_sdk"/dependencies/sbf-tools/llvm/bin/llvm-objcopy "$@"
```

It's probably better to use sbf-tools here. `strip.sh` is still using bpf-tools however.

cc @mvines you created this file originally. The `llvm-native` seems to not exist on new SDK installations. Could you confirm this is safe to change?